### ISSUE TITLE
File Not Found Error with 0.7.1

### ISSUE THREAD

**Issue by sawmonaco** [ga4gh/vrs-python#66]
Hey - I am trying to install the 0.7.1 package (pip install ga4gh.vrs), but I am not able to use the library. It appears that the schema is not being included. 

```
$ pip install ga4gh.vrs
$ python
>>> import ga4gh.core
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/ga4gh/core/__init__.py", line 9, in <module>
    from ._internal.enderef import ga4gh_enref, ga4gh_deref
  File "/usr/local/lib/python3.7/site-packages/ga4gh/core/_internal/enderef.py", line 14, in <module>
    from .identifiers import ga4gh_identify, is_ga4gh_identifier
  File "/usr/local/lib/python3.7/site-packages/ga4gh/core/_internal/identifiers.py", line 39, in <module>
    cfg = yaml.safe_load(open(schema_dir + "/ga4gh.yaml"))
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.7/site-packages/ga4gh/core/_internal/data/schema/ga4gh.yaml'
```

**Comment by ahwagner** [comment]
Thank you for this report! We recently changed our build process to include source dists, and as it turns out setuptools has different behavior with setup.cfg when building sdist vs. bdist, as described in this [Stack Overflow comment](https://stackoverflow.com/a/14159430/3124179) as a "dirty lie".

Fortunately, the solution is very straightforward, just requiring a MANIFEST.in file. Creating a PR with this fix soon.

**Comment by ahwagner** [comment]
This was fixed in #67. Please pip install version 0.7.2 from PyPI.

**Comment by sawmonaco** [comment]
Thanks!
